### DISCUSSION TITLE
Evaluation including large powers of 10?

### DISCUSSION THREAD

**Discussion by OpenShooter** [wxMaxima-developers/wxmaxima#1786] [Q&A]
I am attempting to get an exact solution of 2 simultaneous linear eqns, which contain constants that are large powers of 10. 
I have 2 questions:
Q1) The solution of the resulting quadratic is being shown with the values as integers (e g, 400001000000000400002000000000000001), which I would rather see as the sum of powers of 10 (e g, 4*10^35 + 10^30 + 4*10^20  2*10^15 + 1).
Q2) How can I get Maxima to evaluate the resulting solution, which involves sqrt and constants (e g, (sqrt(400001000000000400002000000000000001)-200000000000000200001000000000000001)/20000000000000020000000000000000000000000) in some easy-to-comprehend form (e g, 1e-5, although this is NOT the solution of the example).
wxMaxima 23.05.1, Windows 10 64-bit.

**Comment by bolimbo** [comment]
An option for Q2:
`(%i5)	float((sqrt(400001000000000400002000000000000001)-200000000000000200001000000000000001)/20000000000000020000000000000000000000000);
(%o5)	-9.999999999999999*10^-6`

**Comment by gunterkoenigsmann** [comment]
Pattern matching doesn't match components of an integer => My guess is that you need a lisp wizard for doing that, who modifies engineering-format.lisp for you. With a bit of luck you might find one at <https://sourceforge.net/p/maxima/mailman/maxima-discuss/>.
